modules/administration-guide/pages/configuring-a-user-namespace.adoc

```diff
@@ -174,9 +174,122 @@ spec:
 ----
 ====
 
+. Create the `Template` below to replicate resources included in the Template to every user {namespace}.
++
+[source,yaml,subs="+attributes,+quotes"]
+----
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: {prod-id-short}-user-namespace-configurator
+  namespace: {prod-namespace}
+  labels:
+    app.kubernetes.io/part-of: che.eclipse.org
+    app.kubernetes.io/component: workspaces-config
+objects:
+  ...
+parameters:
+- name: PROJECT_NAME
+- name: PROJECT_ADMIN_USER
+
+----
++
+The `objects` array can contain any number of resources, including the previously mentioned ConfigMaps, Secrets, and PersistentVolumeClaims.
++
+The `parameters` are optional and define which parameters can be used. Currently, only `PROJECT_NAME` and `PROJECT_ADMIN_USER` are supported. `PROJECT_NAME` is the name of the {prod-short} {namespace}, while `PROJECT_ADMIN_USER` is the {prod-short} user of the {namespace}.
++
+The {namespace} name in objects is omitted and replaced with the user's {namespace} name during synchronization.
```

Comment:
if we want to promote those parameters in Eclipse Che docs we need to cross-link it with OpenShift docs.
I do not see any references in https://docs.openshift.com/container-platform/4.17/rest_api/template_apis/template-template-openshift-io-v1.html